src/api/providers/kilocode-openrouter.ts

```diff
@@ -0,0 +1,79 @@
+import { getModelParams } from ".."
+import { ApiHandlerOptions, openRouterDefaultModelId, openRouterDefaultModelInfo } from "../../shared/api"
+import { DEEP_SEEK_DEFAULT_TEMPERATURE } from "./constants"
+import { OpenRouterHandler } from "./openrouter"
+
+/**
+ * A custom OpenRouter handler that overrides the getModel function
+ * to provide custom model information.
+ */
+export class KilocodeOpenrouterHandler extends OpenRouterHandler {
+	constructor(options: ApiHandlerOptions) {
+		super(options)
+	}
+
+	/**
+	 * Override the getModel function to provide custom model information
+	 */
+	override getModel() {
+		let id
+		let info
+		let defaultTemperature = 0
+		let topP = undefined
+
+		const selectedModel = this.options.kilocodeModel ?? "claude37"
```

Comment:
Isn't this an odd default since the non-openrouter default is clause already too?

Reply:
Good point!